sources/packages/backend/apps/api/src/route-controllers/config/models/get-config.dto.ts

```diff
@@ -1,8 +1,8 @@
-export interface IConfig {
-  auth: IAuthConfig;
+export class ConfigAPIOutDTO {
```

Comment:
The file can be renamed to remove the "get-". It is a really old pattern no longer used.